doc/user/content/cli/_index.md

```diff
@@ -143,15 +158,21 @@ module, then it implicitly applies to all modules. When directives conflict, the
 last directive wins. Materialize will only emit log messages that match at least
 one filter directive.
 
-The module path of a log message reflects its location in Materialize's source
-code. Specifying module paths in filter directives requires familiarity with
-Materialize's codebase and is intended for advanced users. Note that module
-paths change frequency from release to release and are not considered part of
-Materialize's stable interface.
+Specifying module paths in filter directives requires familiarity with
+Materialize's codebase and is intended for advanced users.
 
-The valid levels for a log message are, in increasing order of severity:
-`trace`, `debug`, `info`, `warn`, and `error`. The special level `off` may be
-used in a directive to suppress all log messages, even errors.
+The valid levels for a log message are documented in the [logging
+section](/ops/monitoring/#levels) of the monitoring documentation and may be
+specified in any case. The special level `off` may be used in a directive to
+suppress all log messages, even those at the `error` level.
```

Comment:
I see why you did it this way, but this means that users need to go to two separate pages in order to figure out how to word the directive. I think it would be better to make the Levels section shared content embedded in both pages. If you do that, `off` should be included in a list/table with the other level options.

Reply:
`off` isn't a level though. That's why I did it like this!